controllers/provider-aws/pkg/controller/backupentry/actuator.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backupentry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gardener/gardener-extensions/controllers/provider-aws/pkg/aws"
+	awsclient "github.com/gardener/gardener-extensions/controllers/provider-aws/pkg/aws/client"
+	"github.com/gardener/gardener-extensions/pkg/controller/backupentry"
+
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type actuator struct {
+	backupentry.ProviderActuator
+	client client.Client
+}
+
+func newActuator() backupentry.ProviderActuator {
+	return &actuator{}
+}
+
+func (a *actuator) InjectClient(client client.Client) error {
+	a.client = client
+	return nil
+}
+
+func (a *actuator) Reconcile(ctx context.Context, be *extensionsv1alpha1.BackupEntry) error {
+	return nil
```

Comment:
to-be-implemented, right?

Reply:
Yes. It would be mostly related to backword compatibility.